### DISCUSSION TITLE
Contribution: Improvement ideas to the Splitter -component

### DISCUSSION THREAD

**Discussion by terotests** [primefaces/community#1578] [PrimeReact]
I am currently evaluating PrimeReact for a large software product and I have noticed some issues in Splitter component:

1. No support for ResizeStart event
2. When window loses focus during drag ( blur event ) then the splitter will remain in drag mode causing confusing situation

https://github.com/primefaces/primereact/blob/master/components/lib/splitter/Splitter.js

To solve these I suggest couple of simple changes:

1. Add support for the OnResizeStart event with maybe just the event as parameter
2. Add binding to window "blur" event to cut off the resize

The change 1 would allow sites using IFRAME elements to change pointer-events to none to avoid losing the focus and
then finally the 2 would allow to avoid the issue during CTRL + TAB window switch or other window switch during drag.

**Comment by melloware** [comment]
I would open a GitHub enhancement ticket and provide a PR!
